src/components/Executions/Tables/NodeExecutionChildren.tsx

```diff
@@ -1,132 +1,50 @@
-import { WaitForData } from 'components/common';
-import {
-    RefreshConfig,
-    useDataRefresher,
-    useWorkflowExecution
-} from 'components/hooks';
-import { isEqual } from 'lodash';
-import { Execution, NodeExecutionClosure, WorkflowNodeMetadata } from 'models';
+import { Typography } from '@material-ui/core';
+import * as classnames from 'classnames';
+import { useExpandableMonospaceTextStyles } from 'components/common/ExpandableMonospaceText';
 import * as React from 'react';
```

Comment:
nit: prefer to import "external" (ie react) packages before "local" ones and before components.

Reply:
I would normally agree. I'm using a lint rule/vscode command to enforce the ordering on imports. I think this was the closest I can get, and I had a preference for automated fixing and consistency across all contributors/files. :-/